code/modules/jobs/job_types/station_engineer.dm

```diff
@@ -94,6 +94,27 @@ GLOBAL_LIST_INIT(available_depts_eng, list(ENG_DEPT_MEDICAL, ENG_DEPT_SCIENCE, E
 	else
 		to_chat(M, "<b>You have not been assigned to any department. Patrol the halls and help where needed.</b>")
 
+	var/most_hours_played = 0
+	for (var/mob/plr in GLOB.player_list)
+		var/mob/living/carbon/human/hm = plr
+		if (istype(hm))
+			var/datum/job/J = SSjob.GetJob(hm.job)
+			if (J && J.department_flag == ENGSEC)
+				if ((hm.client.prefs.exp[EXP_TYPE_ENGINEERING] / 60) > most_hours_played)
+					most_hours_played = (hm.client.prefs.exp[EXP_TYPE_ENGINEERING] / 60)
+	
+	//Time to balance
+
+	var/x1 = 10
+	var/x2 = most_hours_played
+	var/x3 = 1000
+	var/y1 = 0.5
+	var/y3 = 120
+
+	var/a1 = (((x2 - x1) * (y3 - y1)) / (x3 - x1)) + y1
```

Comment:
what in the fuck is this magical formula

Reply:
linear interpolation

Reply:
could you put a comment or a better name on what the vars are doing and why. Defines would help.